lib/simp/rake/ci.rb

```diff
@@ -0,0 +1,28 @@
+require 'rake/tasklib'
+require 'simp/ci/gitlab'
+
+module Simp; end
+module Simp::Rake
+  class Ci < ::Rake::TaskLib
+    def initialize( dir )
+       @base_dir = dir
+       define
+    end
+
+    def define
+      namespace :simp do
+
+        desc 'Validate CI configuration'
+        task :ci_lint => [:gitlab_ci_lint] do
+        end
+
+        desc 'Validate GitLab CI configuration'
+        task :gitlab_ci_lint do
```

Comment:
We might want to add an online validator that pings against `gitlab.com` if it's available (and skips if it's not).

https://docs.gitlab.com/ee/api/lint.html